src/goto-checker/bmc_util.h

```diff
@@ -219,6 +221,15 @@ void run_property_decider(
   " --no-self-loops-to-assumptions\n" \
   "                              do not simplify while(1){} to assume(0)\n" \
   " --no-pretty-names            do not simplify identifiers\n" \
+  " --symex-complexity-limit [number]" \
```

Comment:
`[number]` isn't how the arguments are indicated in the other options, e.g. `--max-field-sensitivity-array-size M`. It is confusing to have it in a different format. It makes it look like it's optional (but I don't think it is). :fish: